pkgs/by-name/se/seagoat/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
+  ripgrep,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "seagoat";
+  version = "0.50.1";
+  pyproject = true;
+
+  build-system = [ python3Packages.poetry-core ];
+
+  dependencies = with python3Packages; [
+    appdirs
+    blessed
+    chardet
+    flask
+    deepmerge
+    chromadb
+    gitpython
+    jsonschema
+    pygments
+    requests
+    nest-asyncio
+    waitress
+    psutil
+    stop-words
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-MnRUl6hDfl1NGcV2cXZO+WxVqWUD4TmhP+KkMEYVve8=";
```

The version was changed, but not the hash:

```suggestion
    hash = "sha256-WFAtMmRcoDMoy7TdbenXX4plzihmm7j4dc3yNszMGrM=";
```

You should also change the commit title to reflect the current version.